src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -637,7 +635,10 @@ private void ProduceTagsSynchronously(TaggerContext<TTag> context)
                 {
                     // Otherwise report back on the foreground asap to update the state and let our 
                     // clients know about the change.
-                    RegisterNotification(() => UpdateStateAndReportChanges(newTagTrees, bufferToChanges, newState), 0, cancellationToken);
+                    RegisterNotification(() => UpdateStateAndReportChanges(
```

Comment:
:memo: I really don't like this layout. It makes it look like there are 3 arguments to `UpdateStateAndReportChanges`, and one argument to `RegisterNotification`. The following would be a big improvement:

```csharp
RegisterNotification(
    () => UpdateStateAndReportChanges(newTagTrees, bufferToChanges, newState),
    delay: 0,
    cancellationToken: cancellationToken);
```